### DISCUSSION TITLE
Not able to display new line?

### DISCUSSION THREAD

**Discussion by markwongwkh-code** [i18next/next-i18next#2283] [Q&A]
I found it seems this package can't display new line character in json file. My json file is like this:

`{"key": "a\nb"}`

My expected result is:

a
b

a and b should be in different lines.

But in my webpage, a and b are showing up in the same line.

May I know is it my fault?

Thank you.

**Comment by adrai** [comment]
This is probably related to react and or your UI component...
If not, please provide a minimal reproducible example.
